### PR TITLE
MML-271 Add BiEventType to BiFields

### PR DESCRIPTION
**Pull request by symphony-youness**
Adding a BiEventType makes a difference at the BiFields level of what is a MESSAGEML_MESSAGE_SENT and MESSAGEML_ELEMENT_SENT.
closes: #271

### CODE REVIEW

File: src/main/java/org/symphonyoss/symphony/messageml/bi/BiFields.java [hunk 1]
```diff
@@ -2,84 +2,89 @@
...
+  VALIDATION_MAX("Validation_Max", BiEventType.MESSAGEML_ELEMENT_SENT),
+  VALIDATION_MIN("Validation_Min", BiEventType.MESSAGEML_ELEMENT_SENT),
+  VALIDATION_PATTERN("Validation_Pattern", BiEventType.MESSAGEML_ELEMENT_SENT),
+  VALIDATION_OPTIONS("Validation_Options", BiEventType.MESSAGEML_ELEMENT_SENT),
+  HIGHLIGHTED_OPTIONS("HighlightedOptions", BiEventType.MESSAGEML_ELEMENT_SENT),
+  VALIDATION_STRICT("Validation_Strict", BiEventType.MESSAGEML_ELEMENT_SENT),
+  REQUIRED("Required", BiEventType.MESSAGEML_ELEMENT_SENT),
+  MESSAGE_LENGTH("MessageLength", BiEventType.MESSAGEML_MESSAGE_SENT),
+  ENTITY_JSON_SIZE("EntitiesJSONSize", BiEventType.MESSAGEML_MESSAGE_SENT),
+  FREEMARKER("UseFreeMarker", BiEventType.MESSAGEML_ELEMENT_SENT),
```
Comment (symphony-mariacristina) [inline src/main/java/org/symphonyoss/symphony/messageml/bi/BiFields.java:72]: I think FreeMarker should be MESSAGEML_MESSAGE_SENT, right?

File: src/main/java/org/symphonyoss/symphony/messageml/bi/BiFields.java [hunk 2]
```diff
@@ -2,84 +2,89 @@
...
+  COUNT("Count", null);
```
Comment (symphony-youri) [inline src/main/java/org/symphonyoss/symphony/messageml/bi/BiFields.java:73]: is there a way we can avoid null? (by having a dedicated BiEvenType maybe, like None?

File: src/main/java/org/symphonyoss/symphony/messageml/bi/BiFields.java (hunk 1 above)
Comment (symphony-youness) [reply src/main/java/org/symphonyoss/symphony/messageml/bi/BiFields.java:72]: Yes, nice catch. Thank you @symphony-mariacristina

**Review by symphony-mariacristina — APPROVED** [review]
LGTM!

**Review by symphony-youri — APPROVED** [review]

**Pull request merged**